docker-compose.yml

```diff
@@ -13,6 +13,20 @@ services:
       - ${US_ES_POSTGRES_PORT}:5432
     volumes:
       - entity-service-postgres:/var/lib/postgresql/data
+  mysql:
+    image: mysql
+    container_name: entity-service-mysql
+    restart: always
+    environment:
+      - MYSQL_DATABASE=${MYSQL_DB}
+      - MYSQL_USER=${MYSQL_USER}
+      - MYSQL_PASSWORD=${MYSQL_PASSWORD}
+      - MYSQL_ROOT_PASSWORD=${MYSQL_ROOT_PASSWORD}
+    ports:
+      - ${MYSQL_PORT}:3306
+    volumes:
+      - entity-service--mysql-volume:/var/lib/mysql/data
```

Comment:
Mysql container volume name is incorrect. Should be `/var/lib/mysql`. Check https://hub.docker.com/_/mysql -> Where to Store Data section.